### PR TITLE
fix: various fixes to make simulator work in AWS

### PR DESCRIPTION
**Pull request by nathanielc**
Needed to explicitly set file permissions and use a non local image name for the runner

### CODE REVIEW

File: operator/src/simulation/manager.rs [hunk 1]
```diff
@@ -51,8 +51,8 @@ pub fn manager_job_spec(config: ManagerConfig) -> JobSpec {
                 subdomain: Some("goose".to_owned()),
                 containers: vec![Container {
                     name: "manager".to_owned(),
-                    image: Some("keramik/runner:dev".to_owned()),
-                    image_pull_policy: Some("IfNotPresent".to_owned()),
+                    image: Some("public.ecr.aws/r5b3e0r5/3box/keramik-runner:latest".to_owned()),
+                    image_pull_policy: Some("Always".to_owned()),
```
Comment (smrz2001) [inline operator/src/simulation/manager.rs:55]: Did we want the `Always` here?

File: operator/src/simulation/worker.rs [hunk 2]
```diff
@@ -32,8 +32,8 @@ pub fn worker_job_spec(config: WorkerConfig) -> JobSpec {
             spec: Some(PodSpec {
                 containers: vec![Container {
                     name: "worker".to_owned(),
-                    image: Some("keramik/runner:dev".to_owned()),
-                    image_pull_policy: Some("IfNotPresent".to_owned()),
+                    image: Some("public.ecr.aws/r5b3e0r5/3box/keramik-runner:latest".to_owned()),
+                    image_pull_policy: Some("Always".to_owned()),
```
Comment (smrz2001) [inline operator/src/simulation/worker.rs:36]: And here?

File: operator/src/simulation/manager.rs (hunk 1 above)
Comment (nathanielc) [reply operator/src/simulation/manager.rs:55]: yes, for the `latest` tag we want `Always` or we will not ever see new updates.

File: operator/src/simulation/worker.rs (hunk 2 above)
Comment (nathanielc) [reply operator/src/simulation/worker.rs:36]: Yep, anytime we use `latest` we also want to use `Always`.

**Review by smrz2001 — APPROVED** [review]
🚢

**Review by zachferland — APPROVED** [review]

**Pull request merged**